src/Avalonia.Animation/Timing.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reactive.Linq;
+using Avalonia.Data;
+using Avalonia.Threading;
+
+namespace Avalonia.Animation
+{
+    /// <summary>
+    /// Provides global timing functions for animations.
+    /// </summary>
+    public static class Timing
+    {
+        static ulong _transitionsFrameCount;
+        static PlayState _globalState = PlayState.Run;
+
+        /// <summary>
+        /// The number of frames per second.
+        /// </summary>
+        public const int FramesPerSecond = 60;
+
+        /// <summary>
+        /// The time span of each frame.
+        /// </summary>
+        internal static readonly TimeSpan FrameTick = TimeSpan.FromSeconds(1.0 / FramesPerSecond);
+
+        /// <summary>
+        /// Initializes static members of the <see cref="Timing"/> class.
+        /// </summary>
+        static Timing()
+        {
+            var globalTimer = Observable.Interval(FrameTick, AvaloniaScheduler.Instance);
```

Comment:
Eventually we'll want this to be using the same timer as the renderer loop, but that will be a separate PR. Just leaving a reminder here :)